src/System.Security.Cryptography.X509Certificates/tests/TestData.cs

```diff
@@ -1701,6 +1701,66 @@ internal struct ECDsaCngKeyValues
             "39366d5a59590a734f65735a513d3d0a2d2d2d2d2d454e442043455254494649" +
             "434154452d2d2d2d2d0a").HexToByteArray();
 
+        internal static byte[] ConcatenatedPemFile = ByteUtils.AsciiBytes(
```

Comment:
What happens if some other PEM type is first?
e.g. a PRIVATE KEY / CERTIFICATE multi-PEM?

What happens if some other PEM type is second? (e.g. CERTIFICATE / PRIVATE KEY)

Reply:
Hm. That's a good question. Let me find out.

Reply:
macOS seems to be fine with

```
-----BEGIN RSA PRIVATE KEY-----
MII...
-----END RSA PRIVATE KEY-----
-----BEGIN CERTIFICATE-----
MII...
-----END CERTIFICATE-----
-----BEGIN CERTIFICATE-----
MII...
-----END CERTIFICATE-----
```

It ignores the private key and gives back the first certificate block it encounters. I'll commit a test for it in a bit.

Reply:
@bartonjs Before I add tests, is this something that is likely to work on Windows / openSSL? I'm not in a good spot for the next few days to use a Windows box.

Reply:
> is this something that is likely to work on Windows / openSSL? I'm not in a good spot for the next few days to use a Windows box.

No idea.  If you add the test we'll find out together :smile: (I was just wary of the generic "PEM Sequence" and "Aggregate")